autoload/scratch.vim

```diff
@@ -75,11 +75,11 @@ endfunction
 
 function! s:resolve_height(height)
```

Comment:
Can you change the name of the function to `s:resolve_size` and the argument to `size` (you can use `abs_size` instead of `size` below as well)?